docs/jailer.md

```diff
@@ -45,6 +46,8 @@ jailer --id <id> \
   jailer will use this to join the associated network namespace.
 - When present, the `--daemonize` flag causes the jailer to cal `setsid()` and
   redirect all three standard I/O file descriptors to `/dev/null`.
+- When present, the `--new-pid-ns` flag causes the jailer to exec the provided
```

Comment:
We should also mention that the jailer forks then execs, the result being `firecracker` process will have a different _PID_ than the current `jailer` process.

(Without the flag, `firecracker` runs under the same process/same pid as its jailer).